common/selectors/transactions.ts

```diff
@@ -28,3 +28,7 @@ export function getRecentWalletTransactions(state: AppState): SavedTransaction[]
     return [];
   }
 }
+
+export function getTransactionToRaw(state: AppState) {
```

Comment:
Almost duplicate of `getTo` in `common/selectors/transaction/fields`, you can just extend `getTo` to return `raw` in another function

Reply:
:+1: